tests/applications/flag_control_mixin_test.py

```diff
@@ -0,0 +1,114 @@
+from beamlime.applications.mixins import FlagControlMixin, LogMixin
+
+
+class DummyApp(LogMixin, FlagControlMixin):
+    ...
+
+
+def test_start():
+    app = DummyApp()
+    assert not app._started and app._paused
+    app.start()
+    assert app._started and not app._paused
+
+
+def test_start_twice():
+    app = DummyApp()
+    app.start()
+    assert app._started and not app._paused
+    # app.start()
+    # assert app._started and not app._paused
+
+
+def test_start_twice_paused():
+    app = DummyApp()
+    app.start()
+    app.pause()
+    app.start()
```

Comment:
I don't know if calling `start` if the app is already started should do nothing, as now, or if an exception should be raised?

I guess it depends on what we want to have. Either something that is pretty flexible and will mostly keep running even if it receives commands which are incompatible with its current state, or something that is much more rigid but lets us know immediately if it is receiving the wrong kind of commands...